presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcClient.java

```diff
@@ -42,8 +42,7 @@ default boolean schemaExists(String schema)
 
     List<SchemaTableName> getTableNames(@Nullable String schema);
 
-    @Nullable
-    JdbcTableHandle getTableHandle(SchemaTableName schemaTableName);
+    Optional<JdbcTableHandle> getTableHandle(SchemaTableName schemaTableName);
```

Comment:
This should stay consistent with `io.prestosql.spi.connector.ConnectorMetadata#getTableHandle`

i.e. make this Optional<JdbcTableHandle> when you make `ConnectorMetadata#getTableHandle` return `Optional<ConnectorTableHandle>`

Reply:
That would be the next step, does the order of how things are fixed matter here?

Reply:
This introduces an inconsistency whcih IMO is more harmful that nullability alone.

Reply:
Fixing this first seems fine to me. The inconsistency doesn't seem harmful since `Optional` is part of the return type (you can't forget to check it).